<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>juliaSet</class>
 <widget class="QMainWindow" name="juliaSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>734</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>6500</width>
    <height>7000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Julia's Set</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>690</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drawButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>690</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
   </widget>
   <widget class="QSlider" name="xSlider">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>630</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-5000</number>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="ySlider">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>390</y>
      <width>31</width>
      <height>231</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-5000</number>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="ySliderLabel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>350</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="zSliderLabel">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>590</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X: 0</string>
    </property>
   </widget>
   <widget class="QSlider" name="zSlider">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>390</y>
      <width>31</width>
      <height>231</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="nDagreeslider">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>280</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="sliderPosition">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="nDagreeLabel">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>250</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max iterations</string>
    </property>
   </widget>
   <widget class="QLabel" name="zoomLabel">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>360</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom</string>
    </property>
   </widget>
   <widget class="QSlider" name="cImSlider">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>180</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-10000</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="cReSlider">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>100</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-10000</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="cPattern">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>560</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="ColorPattern">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>530</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Color Pattern</string>
    </property>
   </widget>
   <widget class="QLabel" name="imC">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>160</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Im(c)</string>
    </property>
   </widget>
   <widget class="QLabel" name="reC">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>80</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Re(c)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drawButton_2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>690</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Render</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>764</x>
      <y>32</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Advanced</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>juliaSet</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>586</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
